gamla/functional_async.py

```diff
@@ -1,11 +1,19 @@
 import asyncio
-from typing import Any, AsyncGenerator, Awaitable, Callable, Iterable
+from typing import Any, AsyncGenerator, Awaitable, Callable, Iterable, Type
 
 from gamla import currying, functional_generic
 
 
 def run_sync(f):
-    """Runs a coroutine in a synchronous context, blocking until result arrives."""
+    """
+    Runs a coroutine in a synchronous context, blocking until result arrives.
```

Comment:
the docstring should start immediately with the comment, no newline (silly python)